### PR TITLE
Add recording subcommand

### PR DESCRIPTION
**Pull request by jaredKreppein**
- remove outdated PR for new one
- add `internal/recording/client.go` for method calls
- add `cmd/recording.go` for cli options - call `client.go` and determine output from `http.Response`

### CODE REVIEW

File: cmd/recording.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			Short: "Manage AppMap recordings",
+			Run: func(cmd *cobra.Command, args []string) {
+				cmd.Usage()
+			},
+		}
+		recordingStartCmd = &cobra.Command{
+			Use:   "start [url]",
+			Short: "Start a new AppMap recording session",
+			Args:  cobra.ExactArgs(1),
+      Run: func(cmd *cobra.Command, args []string) {
```
Comment (kgilpin) [inline cmd/recording.go:21]: Always use `gofmt`, which will be applied automatically if you use a tool like VSCode.

**Review by kgilpin — DISMISSED** [review]
@jaredKreppein it looks like there is a big binary `main` accidentally committed. This will need to be eliminated from the Git history, you can ask @apotterri or @dividedmind for help with that if you need it.

File: internal/recording/client_test.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
+package recording
```
Comment (kgilpin) [inline internal/recording/client_test.go:1]: Nice to see all these tests 👍

**Comment by apotterri** [comment]
Just removing the commit that added the binary isn't enough here. It needs to get removed from the history, too. I'll take care of this.

**Review by apotterri — APPROVED** [review]
I rebased the branch and got rid of the executable.

**Pull request merged**